docs/package.json

```diff
@@ -1,15 +1,13 @@
 {
   "private": true,
   "scripts": {
-    "generate": "nuxt generate",
+    "generate": "nuxt generate --force-build",
```

Comment:
Is this to detect changes?

Reply:
It was a remainder from Docus docs, it's fixed in last commit. :)